svix-cli/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "svix-cli"
+description = "A CLI to interact with the Svix API."
+version = "0.1.0"
```

Comment:
Sync the version with the rost of the libs, and please also update the version bump script to update Cargo.toml and cargo.lock like it does for the server. 🙏🏻

Reply:
Sure. I was going to work that as a part of the releasing/cargo dist issue but I can work to get this in here instead.

Reply:
It's fine then, do it as part of the relesae.

Reply:
Added an entry to the script for this. Looks like it should set the version the next time. I manually set it to `1.44.0` in the meantime.